tmk_core/common/eeconfig.h

```diff
@@ -22,7 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <stdbool.h>
 
 #ifndef EECONFIG_MAGIC_NUMBER
-#    define EECONFIG_MAGIC_NUMBER (uint16_t)0xFEEC
+#    define EECONFIG_MAGIC_NUMBER (uint16_t)0xFEED
```

Comment:
```suggestion
#    define EECONFIG_MAGIC_NUMBER (uint16_t)0xFEEB // When changing, decrement this value to avoid future re-init issues
```

Reply:
ty, wasn't sure how that value should be changed exactly.